contributors/devel/api_changes.md

```diff
@@ -927,8 +927,10 @@ The preferred approach adds an alpha field to the existing object, and ensures i
     }
     ```
 
-3. Before persisting the object to storage, clear disabled alpha fields.
-One possible place to do this is in the REST storage strategy's PrepareForCreate/PrepareForUpdate methods:
+3. Before persisting the object to storage, clear disabled alpha fields on create,
+and on update if the existing object does not already have a value in the field.
+This prevents new usage of the feature while it is disabled, while ensuring existing data is preserved.
```

Comment:
This is for the scenario where someone is trying to get things working by toggling knobs, right?

Reply:
it addresses a few scenarios, among them:
- you are in the process of upgrading an HA cluster, have one apiserver at n-1 with the field in alpha, and one at n with the field in beta. something persisted via the new apiserver should still be able to round-trip and pass validation.
- you have upgraded to a release where the field is beta, a severe problem was discovered and you need to disable the feature, and still be able to modify/clean up existing objects with the field persisted while preventing new usages from creeping in

Reply:
Sounds good @liggitt. Thanks for the details.